### PR TITLE
Add Main Hero

### PR DESCRIPTION
**Pull request by chrismekim**
- Adds the main hero and illustration, and a new font for headers
- Note: the site has a max-width of 1400px. I have to talk to Brendon about this hero image to see if it should have the edges straight at all breakpoints, or round. Can replace it during QA if needed! 

Preview:
Screen Shot 2020-09-15 at 1 27 42 PM

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -38,6 +38,37 @@
...
+            src="img/hero-illustration-md.png"
+            srcset="
+              img/hero-illustration-sm.png 240w,
+              img/hero-illustration-md.png 400w,
+              img/hero-illustration-lg.png 720w
+            "
+            sizes="(min-width: 1200px) 720w,
+            (max-width: 768px) 400w,
+            (max-width: 380px) 240w"
+            alt="Illustration of people doing outdoor activies in the mountains"
```
Comment (whoisgina) [inline src/index.html]: "activities"! Sorry, proofreading eyeballs.

File: src/css/main.scss [hunk 2]
```diff
@@ -26,10 +26,32 @@
...
   cursor: pointer;
 }
 
+.button--style-primary {
+  @extend %transition-shorter;
+  background-color: color('blue');
+  text-decoration: none;
+  color: color('white');
+  border-radius: 10px;
+  padding: 0.75rem 1.25rem 0.75rem 1.25rem;
```
Comment (whoisgina) [inline src/css/main.scss]: Tiny nitpick but you can use shorthand here: `padding: 0.75rem 1.25rem;`

**Review by whoisgina — APPROVED** [review]
Looks good! My comments are just suggestions. ✨

File: src/index.html (hunk 1 above)
Comment (chrismekim) [reply src/index.html]: Thank you, good catch!!

**Review by whoisgina — APPROVED** [review]
⚡️

File: src/css/main.scss [hunk 3]
```diff
@@ -26,18 +26,40 @@
...
 button {
   cursor: pointer;
 }
 
+.button--style-primary {
+  @extend %transition-shorter;
+  background-color: color('blue');
+  text-decoration: none;
+  color: color('white');
+  border-radius: 10px;
```
Comment (mokaymokay) [inline src/css/main.scss]: use rem here?

File: src/index.html [hunk 4]
```diff
@@ -38,6 +38,37 @@
...
+    <main>
+      <section class="main-hero flex flex-col inner-content-max-width mxauto">
+        <div class="flex flex-col site-padding-x">
+          <p class="text-body color-gray pb2 sm:pb1_75 md:pb2_5">Hi, we're Seaborne.</p>
+          <h1 class="text-header color-black pb2 sm:pb1_75 md:pb2_5">
+            We're your guides to sustainable business practices.
+          </h1>
+          <a
+            aria-label="Get in touch with Seaborne"
+            class="button--style-primary text-button flex items-center justify-center"
```
Comment (mokaymokay) [inline src/index.html:51]: Maybe add a to-do for the link

**Review by mokaymokay — APPROVED** [review]

File: src/css/main.scss (hunk 3 above)
Comment (chrismekim) [reply src/css/main.scss]: Added in rem :)

File: src/index.html (hunk 4 above)
Comment (chrismekim) [reply src/index.html:51]: Added :)

**Pull request merged**